### ISSUE TITLE
Question about 2d pose estimator

### ISSUE THREAD

**Issue by asw91666** [BrianG13/FLEX#13]
I read your paper with great interest, and thank you very much for sharing your code. I had a few questions after reading your paper.

1) According to your paper, you used Iskakov et al, ICCV 2019 model. Iskakov's proposed model includes an algebraic model and a volumetric model, is it correct that you used the algebraic model as a 2d pose estimator?

2) And did you use ResNet152 backbone and 384x384 image size for 2d pose estimation?

Thank you.

**Comment by BrianG13** [comment]
1. Correct, because the volumetric don't produce any 2D intermediate results, only the algebraic model.
2. Correct, we use their published code and model on their official Git repo.

**Comment by asw91666** [comment]
However, Iskakov's algebraic model is trained using extrinsic parameters. If you used this pretrained model, you should not claim in Table 1 that FLEX is an extrinsic parameter-free approach.

**Comment by BrianG13** [comment]
I agree the Iskakov's algebraic model aggregates the 2D backbone predictions to a 3D voxel using the extrinsics camera parameters, but the 2D backbone is not trained using any of those parameters, so you can use that backbone as a "2D pose estimator from the shelf".
We also show at our paper an ablation study using different 2D pose estimation algorithms
